### PR TITLE
fix(radio-button): update to audit spec and remove center label buttons

### LINKED ISSUE #1926: [Audit] Radio Button: component audit
## Radio button audit

- [x] check focus state width ( it looks like 3px and it should be 2px)

- [x] check distance from label to radio button; should be 8px (looks like more)

- [ ] vertically center radio buttons in component display window (they're riding high now -- this is a website display issue, not a component issue

- [x] @aledavila to be consistent can we say "Radio button label" instead of the word "heading?" We don't use that anywhere else.

- [x] Also, I'm sorry, this is my bad... but looking at the spec... the focus state IS 3px, it's just that the black outline overlays the 3px stroke... in yours, the outline just becomes a 2px blue 60 stroke... but that's not the spec

image

image

- [ ] Not sure if it's just me, but I don't see a focus state in Safari

- [x] Also, we're not offering this to anybody right? It's whack.
image

- [ ] One last point, and this applies to most of the components... can we vertically center them in the component display window? So many of them ride high (not just here but on next)

### PR DESCRIPTION
**Pull request by aledavila**
Closes #1926 

#### Changelog

**Changed**

- labels
- label margin bottom

**Removed**

- radio buttons with labels on top and bottom.

### CODE REVIEW

**Review by jeanservaas — APPROVED** [review]
Looks great! thanks!

File: src/components/radio-button/radio-button.config.js [hunk 1]
```diff
@@ -37,100 +38,94 @@ module.exports = {
...
-      context: {
-        direction: 'top',
-        selectedValue: 'red',
-        group: 'radio-button--vertical',
-        vertical: true,
-        items: items(),
-      },
-    },
-  ],
+  variants: !componentsX
```
Comment (asudoh) [inline src/components/radio-button/radio-button.config.js]: Switching `variants` data statically here doesn't work with our theme switcher. We use `meta.xVersionOnly` and `meta.xVersionNotSupported` for `v10`-only or `v9`-only components/variants.

**Review by asudoh — COMMENTED** [review]
Great to see the update - Just one thing, thanks @aledavila!

File: src/components/radio-button/radio-button.config.js (hunk 1 above)
Comment (aledavila) [reply src/components/radio-button/radio-button.config.js]: Ah thank you. I was struggling to figure out what the opposite of the xVersionOnly was and didn't know we had the xVersionNotSupported. I will change that.

Comment (asudoh) [reply src/components/radio-button/radio-button.config.js]: @aledavila My apologies for the struggle - https://github.com/IBM/carbon-components/blob/v9.68.0/.github/CONTRIBUTING.md#defining-markup-with-conditionals-or-data-interpolations covers those properties, but please don't hesitate to ping me if you stumble at things like this!

**Review by dakahn — APPROVED** [review]
Nice!

**Comment by aledavila** [comment]
@asudoh can you recheck please

File: src/components/radio-button/radio-button.config.js [hunk 2]
```diff
@@ -90,20 +96,6 @@ module.exports = {
         items: items(),
       },
     },
-    {
-      name: 'vertical-bottom',
```
Comment (asudoh) [inline src/components/radio-button/radio-button.config.js:94]: Did we want to remove this variant?

File: src/components/radio-button/radio-button.config.js [hunk 3]
```diff
@@ -118,19 +110,5 @@ module.exports = {
         items: items(),
       },
     },
-    {
-      name: 'vertical-top',
```
Comment (asudoh) [inline src/components/radio-button/radio-button.config.js:122]: Did we want to remove this variant?

**Review by asudoh — COMMENTED** [review]
Thank you for the update @aledavila! - Just a couple questions.

File: src/components/radio-button/radio-button.config.js (hunk 3 above)
Comment (aledavila) [reply src/components/radio-button/radio-button.config.js:122]: Yes both of these were for v10 only but after speaking with @jeanservaas we don't want to offer these variants as they are not correct design wise.

Comment (asudoh) [reply src/components/radio-button/radio-button.config.js:122]: Got it - Thank you for clarifying @aledavila!

**Pull request merged**